automation_tools/__init__.py

```diff
@@ -2055,8 +2034,7 @@ def iso_install(
         return installer_options
 
 
-def product_install(distribution, create_vm=False, certificate_url=None,
-                    selinux_mode=None, sat_version=None,
+def product_install(distribution, certificate_url=None, selinux_mode=None, sat_version=None,
```

Comment:
Removed `create_vm` param as product_install() is no longer handling VM creation